src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CheckDifficultykHybridRule.cs

```diff
@@ -16,6 +16,12 @@ public class CheckDifficultykHybridRule : StakeStoreConsensusRule
         /// <exception cref="ConsensusErrors.BadDiffBits">Thrown if proof of stake is incorrect.</exception>
         public override Task RunAsync(RuleContext context)
         {
+            if (context.SkipValidation)
```

Comment:
This can go in the attribute

Reply:
Didn't you wan't to remove attributes and do it with context.SkipValidation? 
Or now we have both and attribute skips the whole thing and bool SlipValidation is for cases when we want to skip part of the rule?

Reply:
At the moment I left the attribute in place, yeah the flag is useful when a rule has partial skipping.

Reply:
Fixed